hoomd/mpcd/stream.py

```diff
@@ -292,8 +295,8 @@ def _detach_hook(self):
         super()._detach_hook()
 
     @classmethod
-    def _register_cpp_class(cls, geometry, module, cpp_class_name):
-        # we will allow "None" for the force, but we need its class type not its value
+    def _register_cpp_class(cls, geometry, force, module, cpp_class_name):
```

Comment:
`flake8` caught this mistake that the `force` argument got lost in some refactoring commit.